crates/figma_import/src/transform_flexbox.rs

```diff
@@ -143,12 +143,14 @@ fn compute_layout(node: &Node, parent: Option<&Node>) -> ViewStyle {
             LayoutAlignItems::Max => AlignItems::FlexEnd,
             LayoutAlignItems::Min => AlignItems::FlexStart,
             LayoutAlignItems::SpaceBetween => AlignItems::FlexStart, // XXX
+            LayoutAlignItems::Baseline => AlignItems::FlexStart, // Not supported, default to FlexStart
```

Comment:
I really wish we had some way to throw a non-crashing error here to inform the user that they're using an unsupported Figma feature and that it's silently being replaced with a different behavior.

Reply:
We can print an error (though most people wouldn't see it), or think about a way to make it more clear. For now though, I'm just trying to get something quick in.